<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWoScriptSelector</class>
 <widget class="QWidget" name="QWoScriptSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>172</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024000</width>
    <height>274</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTextBrowser" name="log">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string> FileName:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="scriptFile">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReload">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>reload</string>
       </property>
       <property name="icon">
        <iconset resource="qwoterm.qrc">
         <normaloff>:/qwoterm/resource/skin/reload.png</normaloff>:/qwoterm/resource/skin/reload.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowser">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>browser</string>
       </property>
       <property name="icon">
        <iconset resource="qwoterm.qrc">
         <normaloff>:/qwoterm/resource/skin/folder.png</normaloff>:/qwoterm/resource/skin/folder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string> Task:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="task"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string> Entry:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="entry">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string> Arguments:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="args"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnRun">
       <property name="text">
        <string>Run</string>
       </property>
       <property name="icon">
        <iconset resource="qwoterm.qrc">
         <normaloff>:/qwoterm/resource/skin/run.png</normaloff>:/qwoterm/resource/skin/run.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStop">
       <property name="text">
        <string>Stop</string>
       </property>
       <property name="icon">
        <iconset resource="qwoterm.qrc">
         <normaloff>:/qwoterm/resource/skin/stop.png</normaloff>:/qwoterm/resource/skin/stop.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDebug">
       <property name="text">
        <string>Debug</string>
       </property>
       <property name="icon">
        <iconset resource="qwoterm.qrc">
         <normaloff>:/qwoterm/resource/skin/debug.png</normaloff>:/qwoterm/resource/skin/debug.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qwoterm.qrc"/>
 </resources>
 <connections/>
</ui>
